Four Brains/Model/DrumSounds.swift

```diff
@@ -67,9 +80,9 @@ class DrumSounds {
         for beatCardCounter in Range(0...3) {
             for note in Range(0...3) {
                 if beatCardCounter == 0 && note == 0{
-                    rideNoteSequence[0] = wholeBeat.ridePattern[beatCardCounter].beatCardNoteSequence[note]
+                    rideNoteSequence[0] = wholeBeat.getRidePattern()[beatCardCounter].getBeatCardNoteSequence()[note]
```

Comment:
How come we are explicitly setting the 0 index like this?

Reply:
Thanks, I've fixed this. It was unnecessarily overwriting the `0` that I initialized the array with. Updated the initialization to an empty array. No need for the unnecessary `if` statement above.